pycode/examples/simulation/2020_npis_sarscov2_wildtype_germany.py

```diff
@@ -0,0 +1,484 @@
+                xmas = (xmas_date - self.start_date).days
+                dampings.append(contacts_at_home(xmas, 0.0, 0.0))
+                dampings.append(home_office(xmas, 0.4, 0.5))
+                dampings.append(social_events(xmas, 0.4, 0.6))
+                dampings.append(
+                    physical_distancing_home_school(
+                        xmas, 0.0, 0.0))
+                dampings.append(physical_distancing_work_other(xmas, 0.4, 0.6))
+
+                # after christmas
+                after_xmas_date = datetime.date(2020, 12, 27)
+                after_xmas = (after_xmas_date - self.start_date).days
+                dampings.append(contacts_at_home(after_xmas, min, max))
+                dampings.append(home_office(after_xmas, 0.2, 0.3))
+                dampings.append(social_events(after_xmas, 0.6, 0.8))
+                dampings.append(
+                    physical_distancing_home_school(after_xmas, 0.2, 0.4))
+                dampings.append(
+                    physical_distancing_work_other(after_xmas, min, max))
+
+            # local dynamic NPIs
+            dynamic_npis = params.DynamicNPIsInfectedSymptoms
+            local_npis = []
+            # increased from [0.4, 0.6] in Nov
+            local_npis.append(contacts_at_home(0, 0.6, 0.8))
+            local_npis.append(school_closure(0, 0.25, 0.25))  # see paper
+            local_npis.append(home_office(0, 0.2, 0.3))
+            local_npis.append(social_events(0, 0.6, 0.8))
+            local_npis.append(social_events_work(0, 0.1, 0.2))
+            local_npis.append(physical_distancing_home_school(0, 0.6, 0.8))
+            local_npis.append(physical_distancing_work_other(0, 0.6, 0.8))
+            local_npis.append(senior_awareness(0, 0.0, 0.0))
+
+            dynamic_npis.interval = 3.0
+            dynamic_npis.duration = 14.0
+            dynamic_npis.base_value = 100000
+            dynamic_npis.set_threshold(200.0, local_npis)
+
+            # school holidays(holiday periods are set per node, see set_nodes)
+            contacts.school_holiday_damping = damping_helper(
+                0, 1.0, 1.0, lvl_h, typ_s, [loc_s])
+            contacts.dampings = dampings
+
+    def get_graph(self, end_date):
+        model = Model(self.num_groups)
+        self.set_covid_parameters(model)
+        self.set_contact_matrices(model)
+        self.set_npis(model.parameters, end_date)
+
+        graph = secir.ModelGraph()
+
+        scaling_factor_infected = [2.5, 2.5, 2.5, 2.5, 2.5, 2.5]
+        scaling_factor_icu = 1.0
+        tnt_capacity_factor = 7.5 / 100000.
+
+        path_population_data = os.path.join(
+            self.data_dir, "pydata", "Germany",
+            "county_current_population.json")
+
+        s_day = mio.Date(self.start_date.year,
+                         self.start_date.month, self.start_date.day)
+        e_day = mio.Date(end_date.year,
+                         end_date.month, end_date.day)
+        mio.secir.set_nodes(
```

Comment:
Yes. The idea is to use `datetime.date` when the parameter is mostly used in python. However, when we pass this to a cpp funtion this makes problems because we are using our own datatype mio.Date there. 
The implicit casting does not work. I deleted the variables s_day and e_day and directly pass it to the function.